test/basic-querying.test.js

```diff
@@ -625,6 +625,60 @@ describe('basic-querying', function() {
   });
 });
 
+describe('updateOrCreate', function() {
+  var db = getSchema();
+  var Todo;
+
+  before(function setUp(done) {
+    Todo = db.define('Todo', {
+      content: String,
+    });
+    db.automigrate('Todo', done);
+  });
+  afterEach(function tearDown(done) {
+    db.automigrate(done);
+  });
+
+
+  it('creates a model when one does not exist', function(done) {
+    Todo.updateOrCreate({ content: 'a' }, function(err, data) {
```

Comment:
We already have some tests for `updateOrCreate` in [test/manipulation.test.js](https://github.com/strongloop/loopback-datasource-juggler/blob/c9388fa9552cbe41671d96f36c2f22cce1b82811/test/manipulation.test.js#L636), please add your new tests there.